ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -31,7 +31,7 @@ class BaseNodeDisplay(Generic[NodeType]):
     def __init__(self, node: Type[NodeType]):
         self._node = node
 
-    def serialize(self, display_context: "WorkflowDisplayContext", **kwargs: Any) -> JsonObject:
+    def serialize(self, display_context: "WorkflowDisplayContext") -> JsonObject:
```

Comment:
would be nice to keep this API simple - especially since the param that was injected seemed like it could be handled at the consumer (see TryNodeDisplay below)